stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/Controller.kt

```diff
@@ -1,7 +1,7 @@
-package com.stripe.android.ui.core.elements
+package com.stripe.android.uicore.elements
 
 import androidx.annotation.RestrictTo
 
 /** This is a generic controller */
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-sealed interface Controller
+interface Controller
```

Comment:
this cannot be a sealed interface anymore, since some of its implementations live on a different module .